fudge-ratio/src/main/webapp/code/finalScreen.js

```diff
@@ -0,0 +1,57 @@
+const mainText = `
+  <div class="finalMessage">
+    <p class="you" id="result"></p>
+    <p class="others" id="average"></p>
+
+    <div>
+      <button class="active" id="clear">Start another task</button>
+      <button class="active" id="home">Go Home</button>
+    </div>
+  </div>
+  `;
+
+import { get, parent, createFromText } from "./utils.js";
+import { KEYS } from "./save.js";
+import Timer from "./time.js";
+
+const showFinal = () =>
+  new Promise((resolve, _) => {
+    parent.appendChild(createFromText(mainText));
+
+    const startTimer = new Timer().getText();
+    const expected = new Timer(localStorage.getItem(KEYS.EXPECTED_KEY)).seconds;
+    const activity = localStorage.getItem(KEYS.ACTIVITY_KEY);
+    const endTime = localStorage.getItem(KEYS.FINAL_KEY);
+
+    const timer = new Timer(endTime);
+
+    get("clear").onclick = () => {
+      localStorage.clear();
+      location.reload();
+    };
+
+    get("home").onclick = () => location.replace("/");
```

Comment:
I wasn't aware of location.replace, this is a cool way to go back to the homepage without putting the link in html